### DISCUSSION TITLE
SceneViewer aspect ratio is messed up

### DISCUSSION THREAD

**Discussion by hybridherbst** [google/model-viewer#3661] [Q&A]
### Description

I understand that this is a SceneViewer bug and not model-viewer, but since there's no way to submit bugs to SceneViewer I'm submitting this here in the hope it goes to the right people.

I think this is a regression as I'm pretty sure it didn't happen a while back, but AR view in SceneViewer being launched from model-viewer doesn't have proper aspect ratio anymore on a Xiaomi Mi 9.

#### Live Demo
https://near-singularity.glitch.me/ - very visible on the head in the first model

The head is a sphere
![Screenshot_2019-08-07-21-18-57-693_com google android googlequicksearchbox](https://user-images.githubusercontent.com/2693840/62661003-f8e86900-b96f-11e9-89be-ce3fdbc35e39.png)

Phone in landscape, note head is squashed
![IMG_20190807_221035](https://user-images.githubusercontent.com/2693840/62660994-f259f180-b96f-11e9-945a-9e431165e459.png)

Phone in portrait, note head is stretched
![2019-08-07 21 18 41](https://user-images.githubusercontent.com/2693840/62661022-056cc180-b970-11e9-8c67-740e3f1572ad.png)

### Browser Affected

- [x ] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Helios
- [ ] IE
- [ ] Safari

### OS

- [x ] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### Versions

Probably a bug in SceneViewer, not model-viewer

**Comment by cdata** [comment]
Thanks for reporting, I filed this bug internally w/ the Scene Viewer team. I'll report back here if there are any updates.

**Comment by hybridherbst** [comment]
Bump - aspect is still messed up in latest ARCore/SceneViewer updates.

**Comment by elalish** [comment]
@tpsiaki Did this get fixed?

**Comment by AdrianAtGoogle** [comment]
this isn't an issue with the model rendering but with the feed (i.e. the background is similarly squashed and stretched).  We don't have much agency over that -- the system tells us what transform to use to sample from the feed texture and what 3D transform to use to match it.

@hybridherbst can you confirm this issue appears only in SV AR on the Xiaomi Mi 9, and not in ModelViewer's WebXR AR?
